languages/common-lisp/exercises/concept/sameness/.docs/after.md

```diff
@@ -0,0 +1,38 @@
+## Summary
+
+In this exercise you've learned about the four generic equality predicates in Common Lisp: `eq`, `eql`, `equal`, `equalp`. You've learned how they build upon each other and go from a very strict definition of equality (`eq`) to a loose definition of equality (`equalp`).
```

Comment:
The `after.md` document should contain all the information that is in the `introduction.md` file. Part of the reason for this is that we'll make it easy for students to revisit the `after.md` document later, so it will be like a reference document for the involved concepts. The nice thing about the `after.md` document is that you can use it to expand upon things. For a list of things that you could include in the `after.md` document that you wouldn't include in the introduction.md, see https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsaftermd.